app/cdap/components/hydrator/components/TopPanel/NameAndDescription/styles.ts

```diff
@@ -86,7 +86,7 @@ export const PipelineType = styled.div`
 
 export const MetadataLeft = styled.div`
   margin-left: 10px;
-  width: calc(100% - 310px);
+  width: calc(100% - 80px);
```

Comment:
The right edge of name and description input box will grow out of range:
![image](https://user-images.githubusercontent.com/20428112/185668276-344e202e-1bb5-4b28-ab4e-fa817806984d.png)